zstash/parallel.py

```diff
@@ -102,6 +101,7 @@ class PrintQueue(collections.deque):
         """
 
         def __init__(self):
+            # TODO: Created the class independently of the factory method?
```

Comment:
It looks like `namedtuple` creates a new type? Perhaps this code needs to be handled differently to properly add type annotations?

Reply:
You've probably seen this:

>This is expected (although the error message could be better). The NamedTuple constructor isn't meant to be used at runtime in a class constructor; instead you should define your NamedTuple type in the module scope.

https://github.com/python/mypy/issues/7531